docs/self_sign_cert.md

```diff
@@ -0,0 +1,51 @@
+# Accessing private docker registry with self signed certificate
+
+Currently, `jib` do not support docker registries with self signed `https` certificate.
+
+The only way to get `jib` working is to import the self signed certificate into jvm `CA Certificates Keystore`.
+
+## Using KeyStore Explorer
+
+The easiest way to import the self signed certificate into jvm is using the [KeyStore Explorer](http://keystore-explorer.org/).
+
+KeyStore Explorer is an open source GUI replacement for the Java command-line utilities keytool and jarsigner. KeyStore Explorer presents their functionality, and more, via an intuitive graphical user interface.
+
+### Installation
+
+Download and install KeyStore Explorer from [official website](http://keystore-explorer.org/downloads.html)
+
+### Indentify java runtime being used by build tool
+
+#### Maven
+
+Run `mvn --version` and take note on java runtime:
+
+```shell
+$ mvn --version
+Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-18T06:33:14+12:00)
+Maven home: /usr/local/Cellar/maven/3.5.4/libexec
+Java version: 1.8.0_172, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre
+Default locale: en_NZ, platform encoding: UTF-8
+OS name: "mac os x", version: "10.13.6", arch: "x86_64", family: "mac"
+```
+
+On this example the `java runtime` is `/Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre`.
+
+### Import certificate
+
+* Launch `KeyStore Explorer`
```

Comment:
My 2¢: it would be easier to have the user use _Open an existing KeyStore_ and navigate to the `cacerts` file (with default password `changeit`).

Then there are two approaches:
  - if the user has the self-signed certificate, they can import it.
  - if they don't have the self-signed certificate, they can import it from the running service with the _Examine SSL_ option. It allows importing the certificate directly into the keystone.

Reply:
I guess this import process has multiple possible ways to be done.... I describe the one I used